source/Halibut.Tests/ListeningConnectRetryFixture.cs

```diff
@@ -12,10 +14,12 @@ namespace Halibut.Tests
     public class ListeningConnectRetryFixture : BaseTest
     {
         [Test]
-        public async Task ListeningRetriesAttemptsUpToTheConfiguredValue()
+        [LatestClientAndLatestServiceTestCases(testNetworkConditions: false, testWebSocket: false, testPolling:false)]
+        public async Task ListeningRetriesAttemptsUpToTheConfiguredValue(ClientAndServiceTestCase clientAndServiceTestCase)
         {
             TcpConnectionsCreatedCounter tcpConnectionsCreatedCounter = null;
-            using (var clientAndService = await LatestClientAndLatestServiceBuilder.Listening()
+            using (var clientAndService = await clientAndServiceTestCase.CreateTestCaseBuilder()
+                       .As<LatestClientAndLatestServiceBuilder>()
```

Comment:
`As<LatestClientAndLatestServiceBuilder>()` is needed to support `WithEchoService`.

I can remove `As<LatestClientAndLatestServiceBuilder>()` and replace it with `WithStandardServices`.

I think this could be a nicer implementation, but am unsure if this will result in any unforeseen consequences.

FYI, sometimes there are other reasons for needing `As<LatestClientAndLatestServiceBuilder>()`. This is specifically talking about the echo service.

Reply:
Either or :D